### PR TITLE
Added OpenBank function with support for most major banks

### PR DESCRIPTION
**Pull request by StickToTheScript**
Creates a generic function to open most banks. Also tried using two underscores for the utility functions (might require changes, but I'd like to discuss that).

### CODE REVIEW

File: Bank.simba [hunk 1]
```diff
@@ -29,14 +30,196 @@ type
...
+
+{*
+R__GetBankIDs
+~~~~~~~~~~~~~
+Utility function that returns the IDs of the booths.
+*}
+Function R__GetBankIDs(Location: TBankLocation): TIntegerArray;
+begin
+  case Location of
+    TBankLocation.lUMBRIDGE:
```
Comment (Brandon-T) [inline Bank.simba]: Capitalize `L`?

File: Bank.simba [hunk 2]
```diff
@@ -62,9 +245,77 @@ begin
   Result := R_WaitFunction(timeout, 50, @R_BankScreen);
```
Comment (Brandon-T) [inline Bank.simba]: `WaitUntil(R_Bankscreen, 50, timeout)`

**Review by Brandon-T — CHANGES_REQUESTED** [review]

File: Bank.simba (hunk 1 above)
Comment (StickToTheScript) [reply Bank.simba]: Good catch! Fixed!

File: Bank.simba (hunk 2 above)
Comment (StickToTheScript) [reply Bank.simba]: Yea, not a problem! I also took the liberty to change all occurrences of this in the Bank.simba file. As for the other files, I can do those at a different time.

**Review by Brandon-T — APPROVED** [review]

**Pull request merged**